packages/strapi-plugin-content-type-builder/admin/src/InjectedComponents/ContentManager/EditViewLink.js

```diff
@@ -0,0 +1,30 @@
+/**
+ *
+ * EditViewLink
+ *
+ */
+
+import React from 'react';
+import PropTypes from 'prop-types';
+import NavLink from 'components/NavLink';
+
+// Create link from content-type-builder to content-manager
+function EditViewLink(props) {
+  // Retrieve URL from props
+  let url = `${props.getContentTypeBuilderBaseUrl()}${props.getModelName()}`;
```

Comment:
I don't know if this file is going to change a lot in the future but it might be interesting to use a ternary condition with a constant. I'm just asking... I might be wrong on this one ;)